octoprint_mrbeam/templates/wizard/firstrun_end.jinja2

```diff
@@ -1,14 +1,17 @@
 <h3>{{ _('All Done!') }}</h3>
 
-<p>{% trans %}
-    Your Mr Beam II is now all set up and ready to go. Happy
-    lasering!
-{% endtrans %}</p>
+<p>
+    {% trans %}
+        {# Translators: try number 2 #}
+        Your Mr Beam II is now all set up and ready to go. Happy lasering!
+    {% endtrans %}
+</p>
 
 
 <p>
     <iframe width="560" height="315" src="https://www.youtube.com/embed/KKLHeGwc0RY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </p>
 <p>
-    Check out our <a href="https://www.youtube.com/channel/UC8CIMavXllp6S61JgEqSm4A" target="_blank">YouTube channel</a> to find video tutorials and more.
+    {# Check out our <a href="https://www.youtube.com/channel/UC8CIMavXllp6S61JgEqSm4A" target="_blank">YouTube channel</a> to find video tutorials and more. #}
+    {{ _("Check out our %(open)sYouTube channel%(close)s to find video tutorials and more.", open='<a href="https://www.youtube.com/channel/UC8CIMavXllp6S61JgEqSm4A" target="_blank">', close='</a>')|safe }}
```

Comment:
cool! please add this to our documentation! :-)

Reply:
ok